packages/nocodb-sdk/src/lib/globals.ts

```diff
@@ -9,6 +9,37 @@ export enum ViewTypes {
   CALENDAR = 6,
 }
 
+// Central mapping of view types to strings
+const viewTypeStrings: Record<ViewTypes, string> = {
+  [ViewTypes.FORM]: 'form',
+  [ViewTypes.GALLERY]: 'gallery',
+  [ViewTypes.GRID]: 'grid',
+  [ViewTypes.KANBAN]: 'kanban',
+  [ViewTypes.MAP]: 'map',
+  [ViewTypes.CALENDAR]: 'calendar',
+};
+
+// Generate reverse mapping from the original viewTypeStrings
+export const viewTypeToStringMap: Record<ViewTypes, string> = {
+  ...viewTypeStrings,
+};
+
+export const stringToViewTypeMap: Record<string, ViewTypes> = Object.entries(
+  viewTypeStrings
+).reduce((acc, [key, value]) => {
+  acc[value] = Number(key);
+  return acc;
+}, {});
+
+export const ViewTypesNameMap = {
```

Comment:
`viewTypeAlias` as already present in `nc-gui/**/viewUtils.ts`, also the alias map is created in `command-palette.service.ts` both ee and oss file

We have to keep sdk changes as it is and remove from other places